aladdin.sh

```diff
@@ -85,22 +89,54 @@ function check_and_handle_init() {
 }
 
 function set_minikube_config(){
+    local vm_driver=$(jq --arg DEFAULT "$DEFAULT_MINIKUBE_VM_DRIVER" -r '.minikube.vm_driver//$DEFAULT' "$ALADDIN_CONFIG_FILE")
+    local memory=$(jq --arg DEFAULT "$DEFAULT_MINIKUBE_MEMORY" -r '.minikube.memory//$DEFAULT' "$ALADDIN_CONFIG_FILE")
```

Comment:
Is the expectation here that if someone wants to use a different vm driver/memory, they need to update their aladdin config? Currently I think the pattern is for users to not really touch their aladdin config, but I'm open to that pattern being changed.

Reply:
Maybe add support for cpu and disk size while you're in here? i think that will take care of the "most popularly" toggled minikube settings

Reply:
Done.

Reply:
Switched to using aladdin config set/get, rather than looking at a specific file.